src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/config/ExceptionHandler.kt

```diff
@@ -61,6 +63,20 @@ class ExceptionHandler {
       )
   }
 
+  @ExceptionHandler(ConflictException::class)
+  fun handleConflictException(e: Exception): ResponseEntity<ErrorResponse> {
+    log.info("Conflict exception: {}", e.message)
```

Comment:
can we `log.warn` rather than info? as it's something we expect to happen but more important than info.